commander/src/commands/transaction/create/delegate.ts

```diff
@@ -40,6 +41,11 @@ const processInputs = (
 	passphrase?: string,
 ) =>
 	registerDelegate({
+		senderPublicKey: passphrase
+			? getAddressAndPublicKeyFromPassphrase(passphrase).publicKey.toString(
+					'hex',
+			  )
+			: '',
```

Comment:
Should not we change definition of `registerDelegate` to accept public key as Buffer?

Reply:
It should accept, but it's commented on the other side, but this will replaced by https://github.com/LiskHQ/lisk-core/issues/271, so didnt want to change it now.
Also, all the transaction creation functions was for frontend, but since we don't use JSON format, that needs to be redesigned or discontinued when lisk-desktop and mobile start to migrate to 5.0